source/help/registerhelp.md

```diff
@@ -1,48 +1,46 @@
 # Author Registration
 
 To begin registration, visit the [registration form](https://arxiv.org/user/register).
-First, we will ask for your e-mail and you will get to pick a user name
-and password. Your user name must be unique and cannot be changed.  
-Recommendations for arXiv password:
 
--  8 characters or more in length
+Your username must be unique and cannot be changed.  
 
-- May contain upper/lowercase letters, numbers, and one or more underscores (_) - one underscore is required
+arXiv registration also asks for your name and your Organization. If you are unaffiliated you may enter "Indepdendent" as your organization.
+
+Select which archives you are most likely to submit to. This will filter the category options available if you submit an article in the future. 
+
+We will then verify the email for your account.
+
+Please [review account related policies](policies/identity_and_affiliation.md).
 
-- May not contain any other special characters besides underscore - no other punctuation or symbols are allowed (ex. $, ., -,@)
 
-Some successful examples could be: “Pass_Word_123” or “pAssWoRd_123”
+## Account Password
 
-Second, we will ask you for some personal information such as your name,
-your institution (or organization) and which archives you are most
-likely to submit to. We will then verify the email for your account.
+Account password requirements:
 
-*Policy Notes:*
+-  8 characters or more in length
+
+- May contain upper/lowercase letters, numbers, and one or more underscores (_) - one underscore is required
 
--   It is a violation of our policies to misrepresent your identity or
-    organizational affiliation. Claimed affiliation should be current in
-    the conventional sense: e.g., physical presence, funding, e-mail
-    address, etc. Misrepresentation of identity or affiliation, for any
-    reason, is possible grounds for immediate and permanent suspension.
--   Each user may have only one registered arXiv account. If you are
-    attempting to register a new account and are unsure if you already
-    have an account please contact [arXiv user support](https://arxiv.org/support) .
--   Users must comply with arXiv's [Code of conduct](policies/code_of_conduct.md).
+- May not contain any other special characters besides underscore - no other punctuation or symbols are allowed (ex. $, ., -,@)
 
-#### Password
+Some successful password examples could be: “Pass_Word_123” or “pAssWoRd_123”
 
 As with all passwords, we recommend that you keep yours safe. The
 password can be changed from your [personal information page](https://arxiv.org/user/).
 If you forget your password, you can [recover your
 password](https://arxiv.org/user/lost_password) using your registered e-mail address.
 
-#### <span id="emailchange"></span>Email
+<span id="emailchange"></span>
+
+## Email
 
 If you need to change your preferred e-mail address, for example because
```

Comment:
Change the four instances of "e-mail" in this paragraph to "email" to follow other instances on this page (and CMoS).